verl/trainer/ppo/core_algos.py

```diff
@@ -458,18 +458,18 @@ def kl_penalty(logprob: torch.FloatTensor, ref_logprob: torch.FloatTensor, kl_pe
     Returns:
 
     """
-    if kl_penalty == "kl":
+    if kl_penalty == "k1":
```

Comment:
we can simply do `if kl_penalty in ('kl', 'k1'):`
and then remove the verbose changes in verl/trainer/main_ppo.py